tests/assets/testkey.txt

```diff
@@ -0,0 +1,5 @@
+abc def ghi jkl mno pqr stu vwx yz1@3$
```

Comment:
I had made an effort to do so and failed miserably :)

I needed a hardcoded path, since I was testing the file path approach